README.md

```diff
@@ -28,15 +28,20 @@ This workflow is used for running a single SBT task. It can to use for running a
 |------------|-------|--------------------|---------|----------------------|
 | cmd        | 1.0.0 | :exclamation:      | -       | Running command      |
 | java       | 1.0.0 | :heavy_minus_sign: | 8       | _AdoptJDK_ version   |
+| scala      | 1.0.0 | :heavy_minus_sign: | ''      | _Scala_ version      |
 | cache-key  | 1.0.0 | :heavy_minus_sign: | ''      | Key of custom cache  |
 | cache-path | 1.0.0 | :heavy_minus_sign: | ''      | Path of custom cache |
+| env        | 1.0.0 | :heavy_minus_sign: | ''      | Custom ENV vars      |
 
 **How to use**:
 
 ```yaml
 uses: playframework/.github/.github/workflows/sbt.yml@v1
 with:
-  cmd: sbt test
+  cmd: sbt "-Dvar1=$VAR_1" "-Dvar2=$VAR_2" ++$SCALA_VERSION test
+  env: |
+    VAR_1=value
+    VAR_2=value
```

Comment:
Isn't that enough to just call `sbt "-Dvar1=value" ++$SCALA_VERSION test`? 
Why do we need to set vars and let then be resolved?

Reply:
Yes, maybe it's not a good example. 😞 
I'm trying to solve the restriction that in GA we can't set an ENV vars in parent workflow and then use theirs in reusable workflows 😞 
Firstly, currently, the achievement of this is that if `cmd` is not a single line `sbt` command but some script, I don't need to set vars as arguments of this script. For example
```yaml
cmd: "script/big-task.sh"
env: |
  VAR1=value
  VAR2=value
```

Secondly, when GA will support YAML Anchors (https://github.com/actions/runner/issues/1182), we will try to use anchor with env vars in parent workflow and just add them to many jobs instead of copy&paste vars as now (see my draft for main repo https://github.com/playframework/playframework/pull/11142). 🤞